include/micm/util/error.hpp

```diff
@@ -0,0 +1,54 @@
+#if 0
+// Copyright (C) 2023-2024 National Center for Atmospheric Research,
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// This file defines error categories and codes for MICM
+#endif
```

Comment:
why this instead of #pragma once?

Reply:
If we include this in a fortran file, any comments would have to be flagged with `!` instead of `//`. This was just to get the CPP to remove these lines entirely

Reply:
How about not including copyright for this specific file? I think that makes sense that we don't want to duplicate the error file for fortran but it could also be confusing for MICM being standalone CPP project to have this unusual usage of macro to cater for fortran. What do you think?